agent/src/main/java/com/code_intelligence/jazzer/autofuzz/Meta.java

```diff
@@ -344,6 +360,55 @@ static Object consume(FuzzedDataProvider data, Class<?> type, AutofuzzCodegenVis
                                     ", ", "new java.io.ByteArrayInputStream(new byte[]{", "})")));
       }
       return new ByteArrayInputStream(array);
+    } else if (type == Map.class) {
+      if (visitor != null) {
+        // Do not use Collectors.toMap() since it cannot handle null values.
+        visitor.pushGroup("java.util.stream.Stream.of(", ", ",
+            ").collect(java.util.HashMap::new, (map, e) -> map.put(e.getKey(), e.getValue()), java.util.HashMap::putAll)");
```

Comment:
`Collectors.toMap(AbstractMap.SimpleEntry::getKey, AbstractMap.SimpleEntry::getValue)` could be used here. I'm undecided which version I find more readable.

Reply:
Ah, no, sorry. I saw the comment but forgot about it during the review, even though it's right there above the line.